course3/homework-string/my_string.cpp

```diff
@@ -0,0 +1,209 @@
+#include "my_string.h"
+#include <memory>
+
+        MyString::MyString()
```

Comment:
Why indenting the entire file? This doesn't improve readability, only wastes significant portion of precious screen space.